ffv1.md

```diff
@@ -160,7 +160,7 @@ NumBytes is a non-negative integer that expresses the size in 8-bit octets of pa
 
 # General Description
 
-Samples within a plane are coded in raster scan order (left->right, top->bottom). Each sample is predicted by the median predictor from samples in the same plane and the difference is stored see [Coding of the Sample Difference](#coding-of-the-sample-difference).
+Samples within a plane are coded in raster scan order (left->right, top->bottom). Each sample is predicted by the median predictor from samples in the same plane and the difference is stored see [Coding of the Quantized Sample Differences](#coding-of-the-quantized-sample-difference).
```

Comment:
There's a mismatch in the section link.
```
coding-of-the-quantized-sample-difference
vs
Coding of the Quantized Sample Differences
```

I suggest adding an `s` to the end of the link.